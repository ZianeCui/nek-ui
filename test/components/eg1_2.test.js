import Vue from 'vue'
import Test2 from '../../components/Example/eg1/test2.vue'

describe('快照测试', () => {
    let cmp, vm

    beforeEach(() => {
        cmp = Vue.extend(Test2) // Create a copy of the original component
        vm = new cmp({
            data: { // Replace data value with this fake data
                messages: ['Cat']
            }
        }).$mount() // Instances and mounts the component
    })


    it('Test2可以得到期望的html结构', () => {
        expect(vm.$el).toMatchSnapshot()
    })

    //   缺点？

})