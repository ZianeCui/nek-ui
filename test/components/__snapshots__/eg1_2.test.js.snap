// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 Test2可以得到期望的html结构 1`] = `
<div
  id="app"
>
  <ul
    class="test1"
  >
    <li>
      
        Cat
    
    </li>
  </ul>
</div>
`;
